modules/modules.nix

```diff
@@ -216,6 +216,7 @@ let
     (loadModule ./services/taffybar.nix { })
     (loadModule ./services/tahoe-lafs.nix { })
     (loadModule ./services/taskwarrior-sync.nix { condition = hostPlatform.isLinux; })
+    (loadModule ./services/trayer.nix { })
```

Comment:
You'll need to rebase your changes because `loadModule` does not exist anymore in `modules/modules.nix`.